flaky test TestChannelMap

## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

<!-- a step by step guide for reproducing the bug. -->

### 2. What did you expect to see? (Required)

### 3. What did you see instead (Required)

```
=== RUN   TestChannelMap

    misc_test.go:199: channel not closed: blocked

--- FAIL: TestChannelMap (0.00s)

```
### 4. What is your TiDB version? (Required)

<!-- Paste the output of SELECT tidb_version() -->

--- a/util/misc_test.go
+++ b/util/misc_test.go
@@ -195,7 +195,7 @@ func assertChannel[T any](t *testing.T, ch <-chan T, items ...T) {
 	select {
 	case item, ok := <-ch:
 		assert.False(t, ok, "channel not closed: more item %v", item)
-	default:
+	case <-time.After(50 * time.Microsecond):
 		t.Fatal("channel not closed: blocked")
 	}
 }

